tests/unit/test_statistics_cmds.py

```diff
@@ -0,0 +1,344 @@
+      'args': ['status', '-h']},
+     {'stdout': STATISTICS_STATUS_HELP_LINES,
+      'rc': 0,
+      'test': 'innows'},
+     None, OK],
+
+    # Test the local statistics commands against a mock.  The mock
+    # returns some data. Note tht the Opt Time in responses masked out in result
+    # because there are not fixed values.
+    ['Verify Statistics auto option against mock executes op and displays',
+     {'general': ['-T', 'auto'],
+      'cmdgrp': 'qualifier',
+      'args': ['get', 'Key']},
+     {'stdout': ['Qualifier Key : boolean = false',
+                 'Op    Exc     Op Time(S)  Operation',
+                 '1      0 ', 'GetQualifier',
+                 '10  0', 'SetQualifier'],
+      'rc': 0,
+      'test': 'innows'},
+     SIMPLE_MOCK_FILE, OK],
+
+    ['Verify Statistics auto option statistics reset command',
+     {'general': ['--timestats', 'auto'],
+      'cmdgrp': 'statistics',
+      'args': ['reset']},
+     {'stdout': ['Op     Exc    Op Time(S)'],
+      'rc': 0,
+      'test': 'innows'},
+     SIMPLE_MOCK_FILE, OK],
+
+    ['Verify Statistics auto option against mock executes op and displays',
+     {'general': ['--timestats', 'auto'],
+      'cmdgrp': 'qualifier',
+      'args': ['get', 'Key']},
+     {'stdout': ['Qualifier Key : boolean = false',
+                 'Op    Exc     Op Time(S)  Operation',
+                 '1      0 ', ' GetQualifier'],
+      'rc': 0,
+      'test': 'innows'},
+     SIMPLE_MOCK_FILE, OK],
+
+
+    ['Verify Statistics auto option against mock executes op and displays',
+     {'general': ['-T', 'auto'],
+      'cmdgrp': 'qualifier',
+      'args': ['get', 'Key']},
+     {'stdout': ['Qualifier Key : boolean = false',
+                 'Op    Exc     Op Time(S)  Operation',
+                 '1      0 ', 'GetQualifier'],
+      'rc': 0,
+      'test': 'innows'},
+     SIMPLE_MOCK_FILE, OK],
+
+
+    ['Verify Statistics status statistics off',
+     {'general': [],
+      'cmdgrp': 'statistics',
+      'args': ['status']},
+     {'stdout': [],
+      'stderr': ['Error: Server connection failed'],
+      'rc': 1,
+      'test': 'innows'},
+     SIMPLE_MOCK_FILE, OK],
+
+
+    ['Verify use of -T cmd and statistis reset.',
+     {'general': ['--mock-server', SIMPLE_MOCK_FILE_PATH, '-T', 'cmd'],
+      # args not allowed in interactive mode
+      'stdin': ['class enumerate --di --no',
+                'class enumerate --di --no',
+                'statistics reset',
+                'class enumerate --di --no',
+                'statistics show', ],
+      'cmdgrp': None,
+      },
+     {'stdout': [r'1 *0[ 0-9.]*EnumerateClassNames'],
+      'rc': 0,
+      'test': 'regex'},
+     None, OK],
+
+    # TODO tests not done
+    # 1. Tests against a real server that returns statistics to show
+    #    the server times in the table.
+    # 2. Tests of statistics server-on, server-off (requires a WBEM server)
+    # 3. Test of statistics commands with no current server (Should fail)
+
+]
```

Comment:
These test cases result in a coverage of 60.82% for the _cmd_statistics.py file. That value of course pulls down the overall coverage.

Have a look at the [coveralls.io results](https://coveralls.io/builds/36929966/source?filename=pywbemtools%2Fpywbemcli%2F_cmd_statistics.py) to find what the test cases do not cover.

Reply:
Agree.  Am adding one more local test on server-on and server-off.